e2e/cypress/integration/messaging/single_image_thumbnail_spec.js

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging @smoke
+
+import * as MESSAGES from '../../fixtures/messages';
+
+describe('Message', () => {
+    before(() => {
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup({loginAfter: true}).then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-T1796 Standard view: Show single image thumbnail', () => {
+        verifySingleImageThumbnail({mode: 'Standard'});
+    });
+
+    it('MM-T1797 Compact view: Show single image thumbnail', () => {
+        verifySingleImageThumbnail({mode: 'Compact'});
+    });
```

Comment:
These tests are unstable. So, extracted the tests and move into its own spec file and not mark as `@prod`.